lms/djangoapps/django_comment_client/base/tests.py

```diff
@@ -1642,6 +1642,40 @@ def test_team_events(self, view_name, event_name, view_data, view_kwargs, mock_r
         self.assertEqual(name, event_name)
         self.assertEqual(event['team_id'], team.team_id)
 
+    @ddt.data(
+        ('vote_for_thread', 'thread_id', 'thread'),
+        ('undo_vote_for_thread', 'thread_id', 'thread'),
+        ('vote_for_comment', 'comment_id', 'response'),
+        ('undo_vote_for_comment', 'comment_id', 'response'),
+    )
+    @ddt.unpack
+    @patch('eventtracking.tracker.emit')
```

Comment:
@dianakhuang Thanks for the pointer, good to know.  I briefly looked into porting the current code over, but that would be a bit cumbersome, since I would have to port over all existing test cases (and the current assertion lists are far from being complete, so they would need to be updated as well).